src/request/request-tab.js

```diff
@@ -35,9 +35,16 @@ module.exports = {
 
 // TODO: Need to come up with a better self registration process
 require('./components/request-detail-panel-execution');
-require('./components/request-detail-panel-webservices');
```

Comment:
Yep, it *was* but was brought back to life due to a merge conflict.  It should be dead again.